docs/dev/guide/contract-verification.md

```diff
@@ -62,44 +67,38 @@ To verify the contract, enter the following details;
 - Contract address: The address provided must match with `0x` address supplied during contract creation.
 - Contract name: The name must be identical to the name provided in the contract.
 - Optimization: This checks whether or not optimization was used while compiling the contract, If you enabled optimization during compilation, check **Yes** otherwise select **No**
-- Compiler version: This specifies the exact version of the compiler used to compile a Smart Contract. Click on the dropdown to specify the compiler version used.
+- Compiler version: This specifies the exact version of the compiler used to compile a smart contract. Click on the dropdown to specify the compiler version used.
   
 <br>
 
 ![Compiler version!](../../assets/images/compiler-version.png "compiler version")
 
 - Enter the Solidity Contract code: Copy the code from your editor and paste in the text area.
 <br>
-**Note:** You may need to flatten your solidity code if it utilizes a library or inherits dependencies from another contract. 
+**Note:** You may need to flatten your Solidity code if it utilizes a library or inherits dependencies from another contract. 
 <br> 
-We recommend using the [truffle flattener](https://github.com/NomicFoundation/truffle-flattener) or the [POA solidity flattener](https://github.com/poanetwork/solidity-flattener).
+We recommend using the [truffle flattener](https://github.com/NomicFoundation/truffle-flattener) or the [POA Solidity flattener](https://github.com/poanetwork/Solidity-flattener).
 
-- Constructor arguments: If Constructor Arguments are required by the contract, you should add them here, but in [ABI hex encoded form](https://solidity.readthedocs.io/en/develop/abi-spec.html).
+- Constructor arguments: If Constructor Arguments are required by the contract, you should add them here, but in [ABI hex encoded form](https://Solidity.readthedocs.io/en/develop/abi-spec.html).
   
 <br>
 
-**Note:** To obtain the Constructor arguments, after deploying your contract, include this 
-<br>
-
-```
-console.log("constructor args:" + greeterContract.interface.encodeDeploy([greeting]));
+### How can you obtain your constructor arguments?
 
-``` 
+> **Note:** The easiest way to get the constructor argument data is to write it by yourself to the console while deploying.
 
-For example;
+For example, if using our [tutorial](./hello-world.md), you can find line:
 
-```typescript
-// Deploy this contract. 
-// `greeting` is an argument for contract constructor.
+`const greeterContract = await deployer.deploy(artifact, [greeting]);` <br>
```

Comment:
This is obviosuly debatable, but I think we should avoid having stuff like `<br>` in our documentation if we can achieve the same goals with pure markup.

Instead of writing code snippet with "`" and then doing "\<br>", you could do a block snippet ```js

It would achieve the same goal:

- The snippet is on a separate line
- As a bonus it gets correct syntax highlighting, which makes it easier to read